### ISSUE TITLE
IE11 throws a "Unable to get property 'resolve' of undefined or null" error

### ISSUE THREAD

**Issue by taesup** [wojtekmaj/react-pdf#489]
**Before you start - checklist**

 - [x] I followed instructions in documentation written for my React-PDF version
 - [x] I have checked if this bug is not already reported
 - [x] I have checked if an issue is not listed in [Known issues](https://github.com/wojtekmaj/react-pdf/wiki/Known-issues)

**Description**

It looks like the version of pdf.js currently being used with react-pdf (2.1.266) has a bug related to poly-filling promises for IE11. After playing around with it a bit, the pdf.js demo site seems to load fine with pdf.js@2.4.142 so it looks like they have fixed that bug in that version at least. Requesting to update the pdf-dist dep to latest (2.3.200) in hopes that the IE11 bug is fixed by that point as well. 

**Steps to reproduce**

1. Using IE11 try to render any pdf file

**Expected behavior**

PDF file should render with no errors

**Additional information**
N/A

**Environment**

 - Browser (if applicable) [e.g. Chrome 57, Firefox 59]: IE11
 - React-PDF version [e.g. 3.0.4]: 4.1.0
 - React version [e.g. 16.3.0]: 16.3.2
 - Webpack version (if applicable) [e.g. 4.16.2]: 4.30.0

**Comment by wojtekmaj** [comment]
Hi,
can you share an error stack? It'll help me locate the problem.

**Comment by chrisphillers** [comment]
Hi Wojek,
What specifically do you need? 

Console gives me:
SCRIPT1002: Syntax error
src_components_Repl_03a039ea6244c178303d39abc507e1f2-200487639f4b6fdd1335.js (1,355812)
SCRIPT1002: Syntax error
repl.js (1,1050845)

Not too wordy is it.

When visiting your demo (I get the same issue on my site if your repo is installed)
https://react-pdf.org/repl

**Comment by wojtekmaj** [comment]
Closing as IE11 is no longer supported by latest version of React-PDF.
